### ISSUE TITLE
Improvement for Announce Relationship

### ISSUE THREAD

**Issue by steph-ieffam** [coar-notify/notify.coar-repositories.org#26]
About the Announce Relationship there seems to be some improvements that can be implemented to simplify the handling of the LDN.
The current LDN for **Announce Relationship** is the following:

```
{
  "@context": [
    "https://www.w3.org/ns/activitystreams",
    "https://purl.org/coar/notify"
  ],
  "actor": {
    "id": "https://research-organisation.org",
    "name": "Research Organisation",
    "type": "Organization"
  },
  "context": {
    "id": "https://another-research-organisation.org/repository/datasets/item/201203421/",
    "ietf:cite-as": "https://doi.org/10.5555/999555666",
    "ietf:item": {
      "id": "https://another-research-organisation.org/repository/datasets/item/201203421/data_archive.zip",
      "mediaType": "application/zip",
      "type": [
        "Article",
        "sorg:Dataset"
      ]
    },
    "type": "sorg:AboutPage"
  },
  "id": "urn:uuid:94ecae35-dcfd-4182-8550-22c7164fe23f",
  "object": {
    "as:object": "https://another-research-organisation.org/repository/datasets/item/201203421/",
    "as:relationship": "http://purl.org/vocab/frbr/core#supplement",
    "as:subject": "https://research-organisation.org/repository/item/201203/421/",
    "id": "urn:uuid:74FFB356-0632-44D9-B176-888DA85758DC",
    "type": "Relationship"
  },
  "origin": {
    "id": "https://research-organisation.org/repository",
    "inbox": "https://research-organisation.org/inbox/",
    "type": "Service"
  },
  "target": {
    "id": "https://another-research-organisation.org/repository",
    "inbox": "https://another-research-organisation.org/inbox/",
    "type": "Service"
  },
  "type": [
    "Announce",
    "coar-notify:RelationshipAction"
  ]
}
```
regarding the context object I think some data is hard to find/retrieve and prob. is useless for any scenario.
In the following object any other info exception from **id** is quite hard to retrieve and moreover I see no improvement in including these additional info in the context section.
For other scenario (solicited ones) this is quite useful since the sender includes some well-known information about their item. In the case of an unsolicited scenario where my repository is supposed to notify the relationship between my local item and an external item we usually don't have all the following information especially about the files. It would be basically really hard to retrieve/collect these additional information that the receiver is not even using considering that the receiver already knows everything about their own items and do not need these information.

```
  "context": {
    "id": "https://another-research-organisation.org/repository/datasets/item/201203421/",
    "ietf:cite-as": "https://doi.org/10.5555/999555666",
    "ietf:item": {
      "id": "https://another-research-organisation.org/repository/datasets/item/201203421/data_archive.zip",
      "mediaType": "application/zip",
      "type": [
        "Article",
        "sorg:Dataset"
      ]
    },
    "type": "sorg:AboutPage"
  }
```

**Comment by paulwalk** [comment]
I think that I agree with all that you say here except for " and prob. is useless for any scenario.". I can imagine scenarios where the `context` might be useful - for example if the *AnnounceRelationship* is at the end of a longer exchange of a series of notifications about some resource.

In any case, we have specified the inclusion of `context` OPTIONAL, so there is no problem in leaving it out, and we have specified its  sub-properties as RECOMMENDED so, again, some or all of these can be omitted (of course `id` is REQUIRED in any case)
